documentation/storybook.css

```diff
@@ -12,4 +12,25 @@
 
 .sbdocs-content {
   max-width: 1200px !important;
+}
+
+:is(.sbdocs-table, .docblock-argstable) {
```

Comment:
I think I may have asked about this before, but what's the benefit of using the is selector in this use case?

Reply:
Ah…technically we don't need it here, but down below it's super handy as this:

```css
:is(.sbdocs-table, .docblock-argstable) :is(th, td) {
```

is much easier to write out than this:

```css
.sbdocs-table th, .sbdocs-table td, .docblock-argstable th, .docblock-argstable td {
```

ref: https://developer.mozilla.org/en-US/docs/Web/CSS/:is